winlogbeat/_meta/fields.common.yml

```diff
@@ -184,7 +184,6 @@
 
     - name: xml
       type: keyword
-      type: text
```

Comment:
@andrewkroh Not sure which one was the original intention.

Reply:
It was `text` because the value is very long and similar to `message`. If you change it to `keyword` then ignore_above needs to be increased from our default.

Reply:
If full text indexing is deemed important here, we should add it as a multi-field at `eventlog.xml.text`.

Increasing ignore_above wouldn't help much, I would think.

Reply:
I'm also good with keeping this as text (I think at the moment it is text because yaml takes the last definition). I don't really see how this field would be used for aggregations or exact match queries.